app/assets/stylesheets/home/components/prices/prices.scss

```diff
@@ -0,0 +1,128 @@
+@import '../variables';
+
+.prices-head {
+    @extend %flex-column;
+    margin-top: 60px;
+    }
+
+    .prices-text {
+        font-size: 60px;
+        color: $text-color;
+        margin-left: 10px;
+        margin-right: 10px;
+        text-align: center;
+
+         p {
+            font-size: 20px;
+        }
+      }
+
+.prices-table {
+    display:inline-block;
+    width:100%;
+    margin-top:50px
+   }
+   
+   /*Boxes*/
+   
+   .prices-box {
+    display:inline-block;
+    background-color:#f3f3f3;
+    width:100%;
+    max-width:500px;
+    border-radius:10px;
+    margin-bottom:50px;
+    box-shadow: 0px 2px 5px rgb(107, 106, 106)
+   }
+   
+   @media screen and (min-width:768px) {
```

Comment:
Hmm, there are many lines that are wrongly indented in this file. Please fix the indentation for the whole file. 

Remember, we usually indent using two spaces per level of indentation. So if we have code like this:

```scss
.selector1 { 
width: 100%;
height: 50%;

.selector2 { margin-top: 15px; }
}
.selector3{ padding: 5em; }
```

then it should look like this:

```scss
.selector1 { 
  width: 100%; 
  height: 50%;
  
  // Notice this selector is nested inside .selector1, so its indentation is deeper
  .selector2 { 
    margin-top: 15px;
  }
}

// this selector is not nested inside .selector1, so its indented at the same level
.selector3 {
  padding: 5em;
}
```

Whitespaces may not be important for computers, but they're really helpful for humans reading the code.

Reply:
By the way, you can always ask your editor to indent correctly for you. Since you're using VS Code, do `CTRL + SHIFT + P` and look for `Reindent lines`.